### PR TITLE
DP-8712  support 2 executives in about row

### PR DESCRIPTION
**Pull request by clair0917**
## Description
Adds a variation of the about section with 2 cards

## Related Issue / Ticket

- [JIRA issue DP-8712](https://jira.state.ma.us/browse/DP-8712)

## Steps to Test

1. The new pattern variation can be seen at /?p=organisms-about-section-two-officials

## Screenshots
screen shot 2018-04-25 at 12 23 13 pm

## Additional Notes:
No backstop files added because nothing changed

### CODE REVIEW

File: styleguide/source/_patterns/03-organisms/by-author/about-section.twig [hunk 1]
```diff
@@ -27,5 +27,18 @@
       {% include "@molecules/icon-links.twig" %}
     {% endblock %}
   </div>
+
+  {% if about.secondary_person %}
+    <div class="ma__about-section__people">
+      <div class="ma__about-section-person primary">
+        {% set person = about.person %}
```
Comment (tnathanjames) [inline styleguide/source/_patterns/03-organisms/by-author/about-section.twig]: Can a block be added here, maybe primaryPersonCard?

File: styleguide/source/_patterns/03-organisms/by-author/about-section.twig [hunk 2]
```diff
@@ -27,5 +27,18 @@
...
+        {% include "@organisms/by-author/person-card.twig" %}
+      </div>
+      <div class="ma__about-section-person secondary">
+        {% set person = about.secondary_person %}
```
Comment (tnathanjames) [inline styleguide/source/_patterns/03-organisms/by-author/about-section.twig]: Can a block be added here, maybe secondaryPersonCard?

File: styleguide/source/_patterns/03-organisms/by-author/about-section.md [hunk 3]
```diff
@@ -25,5 +25,9 @@ about: {
   socialLinks: [{
     type: icon links
   }]
+
+  people: [{
```
Comment (isaacchansky) [inline styleguide/source/_patterns/03-organisms/by-author/about-section.md]: this looks like it conflicts with the JSON - we should have `person` and `secondary_person` here, right?

**Review by isaacchansky — CHANGES_REQUESTED** [review]

File: styleguide/source/_patterns/03-organisms/by-author/about-section.md (hunk 3 above)
Comment (clair0917) [reply styleguide/source/_patterns/03-organisms/by-author/about-section.md]: you are absolutely right. I forgot to update the file when I changed plans. Good catch

**Review by isaacchansky — APPROVED** [review]

**Pull request merged**